deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml

```diff
@@ -13,3 +13,9 @@ spec:
     spec:
       etcdCluster:
         size: 1
+  - name: ibm-cert-manager-operator
+    spec:
+      enableWebhook: true
+      imageRegistry: quay.io/opencloudio
+      pullSecret:
```

Comment:
can you remove this? instead you can put this into cert-manager operator CSV file.

Later when you want to change something, you don't need to notify meta-operator.

Reply:
Do I put `certManager: {}` instead of all of this instead?